packages/devtools_app/lib/src/screens/vm_developer/vm_function_display.dart

```diff
@@ -39,8 +39,9 @@ class VmFuncDisplay extends StatelessWidget {
     String? boolYesOrNo(bool? condition) {
       if (condition == null) {
         return null;
+      } else {
```

Comment:
you can return the `else` statement here since we return from the `if`

Reply:
Done